src/PDFDoc.php

```diff
@@ -70,7 +70,7 @@ class PDFDoc extends Buffer {
     protected $_buffer = "";
     protected $_backup_state = [];
     protected $_certificate = null;
-    protected $_signature_ltv = null;
```

Comment:
how if we want to perform ltv with default way (just set ltv true) without add optional ocsp, crl, and issuer?

Reply:
Just do
```php
$obj->set_ltv(); // nor arguments creates an array with default values, so `_signature_ltv_data` is not null
```